docs/installation-and-operations/system-requirements/README.md

```diff
@@ -12,10 +12,51 @@ provide any official support for them.
 
 ## Server
 
-### Hardware
+The server hardware requirements should be rougly the same for both the packaged installation and docker (both all-in-one container and compose).
 
+### Minimum Hardware Requirements
+
+* __CPU:__ Quad Core CPU (>= 2ghz)
 * __Memory:__ 4096 MB
-* __Free disk space:__ 4 GB
+* __Free disk space:__ 20 GB
+
+This is for a single Server running OpenProject for up to 200 total active users. Depending on your number of peak users, these requirements might vary drastically.
+
+### General Requirements
+
+Generally speaking you will need more CPUs (the faster the better) and more RAM with an increasing number of users.
+Technically this only really depends on the number of __peak__ users. No matter if you have 1000 or only 100 users overall, if there only ever 20 users working at the same time, the CPU & RAM required will be the same.
+Still, the total number of users is a good general indicator of how much resources you will need.
+
+It's not enough to simply have more resources available, however. You will have to make use of them too.
+By default OpenProject has 4 so called web workers and 1 background worker.
+If there are more users you will need more web workers and eventually also more background workers.
+
+The database will need resources as well, and this, too, will increase with the number of users.
+There may come a point where you will have to make configuration changes to the database and/or use an external database, but for most cases the default database setup should be enough.
+
+Using a rough estimate we can give the following recommendations based on the number of users.
+
+| Users | CPU cores | RAM in GB  | web workers | background workers | Diskspace in GB |
+|-------|-----------|------------|-------------|--------------------|-----------------|
+| 200   | 4         | 4          | 4           | 1                  | 20              |
+| 500   | 8         | 8          | 8           | 2                  | 40              |
+| 1500  | 16        | 16         | 16          | 4                  | 80              |
```

Comment:
Because one user request can lead to multiple browser requests I'd increase the number of web workers at least for the 200 users scenario. 
I think I'd start with 4 workers for up to 20 (or maybe even 50) users and go to 8 for 200. 16 for 500 and 32 for 1500. The factor of requiring multiple actual requests per user triggered request will probably balance out better the more users are involved. 
It is probably better to have higher requirements then to later have to deal with performance complaints.

Reply:
Previously we've been talking about total or concurrent users. Is this table targeted at total or concurrent users? For concurrent, the values would be much higher and I'd stop at 500 / 32gig

Reply:
The table is aimed at total users. I should perhaps amend the heading in that case. Concurrent users is nice in theory practically people will find that hard to answer. At least hard than total users.